### PR TITLE
Optimize load tests by streaming reports

### PR DESCRIPTION
**Pull request by mendess**
- Improve usablity of dapf around vdaf configs
- Generate reports dinamically in load tests

### CODE REVIEW

File: daphne_service_utils/dapf/src/acceptance/mod.rs [hunk 1]
```diff
@@ -878,3 +832,99 @@ pub fn now() -> Now {
...
+/// An iterator that yields the size of chunks based on a length and a maximum chunk size.
+///
+/// # Example
+/// ```
+/// use dapf::acceptance::Chunker;
+///
+/// assert_eq!(Chunker::of_size(200, 51).collect::<Vec<_>>(), [51, 51, 51, 47]);
+/// ```
+#[derive(Debug, Clone, Copy)]
+pub struct Chunker {
```
Comment (Noah-Kennedy) [inline daphne_service_utils/dapf/src/acceptance/mod.rs:845]: Where is this used?

File: daphne_service_utils/dapf/src/acceptance/report_generator.rs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+impl ReportGenerator {
+    pub(super) fn new(
+        test_task_config: &TestTaskConfig,
+        reports_per_batch: usize,
+        measurement: &DapMeasurement,
+        version: DapVersion,
+        now: Now,
+    ) -> Self {
+        let (tx, rx) = mpsc::channel(4);
+        thread::spawn({
```
Comment (Noah-Kennedy) [inline daphne_service_utils/dapf/src/acceptance/report_generator.rs:67]: Why not just use `rayon::spawn` on each job?

File: daphne_service_utils/dapf/src/acceptance/mod.rs (hunk 1 above)
Comment (mendess) [reply daphne_service_utils/dapf/src/acceptance/mod.rs:845]: It's used in this same file to produce an iterator to split the number of report among jobs. Tbh there's probably a shorter way to write this and I'm just suffering from acute skill issue :rofl:

Comment (Noah-Kennedy) [reply daphne_service_utils/dapf/src/acceptance/mod.rs:845]: There probably is, but never let the perfect be the enemy of the good.

The jank will do just fine.

**Review by cjpatton — APPROVED** [review]
Changes to the `daphne` crate look good to me. As for the changes to `dapf`, consider my approval a stamp to unblock you. At a high level it seems like a reasonable way to speed up report generation. The changes to the CLI are a little clunky but I don't have any better ideas.

I would recommend adding some code comments to guide the reader. I'm thinking in particular of the `ReportGenerator` module: what we're doing here is simulating a bunch of Clients generating reports, which is just really really expensive to do on a single machine. In production, this will be spread across actual Client devices and we expect the cost per-device to be low.

**Pull request merged**